molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl

```diff
@@ -184,7 +184,7 @@
             </#if>
             <div class="navbar-header">
                 <#list menu.items as item>
-                    <#if item.type != "MENU" && item.name == "Home" && app_settings.logoNavBarHref?has_content>
+                    <#if !item.menu && item.label == "Home" && app_settings.logoNavBarHref?has_content>
```

Comment:
This is because of how the bean specification works, the name of the boolean property described by `isMenu()` is simply `menu`. I tend to prefer using the property names in freemarker, but perhaps here the complete getter is clearer. Switching to item.isMenu() here and elsewhere.